src/main/java/com/cloudbees/plugins/credentials/CredentialsProvider.java

```diff
@@ -834,3 +839,15 @@ public void remove() {
     }
 
 }
+
+/**
+ * Comparator to alphabetically sort credentials drop down list in ascending order by Credential Name.
+ *
+ */
+class CredentialsNameComparator implements Comparator<Credentials>, java.io.Serializable{
+	private static final long serialVersionUID = 1;
```

Comment:
My eclipse gave me a note about it so I wasn't sure if findbugs would also mention it.

whoops i'll fix whitespace.